coordinator/src/main/scala/filodb.coordinator/client/FiloKryoSerializers.scala

```diff
@@ -82,23 +81,28 @@ class PartitionInfoSerializer extends KryoSerializer[PartitionInfo] {
 
 class PromQlQueryParamsSerializer extends KryoSerializer[PromQlQueryParams] {
   override def read(kryo: Kryo, input: Input, typ: Class[PromQlQueryParams]): PromQlQueryParams = {
-    val config = ConfigFactory.parseString(input.readString())
     val promQl = input.readString()
     val start = input.readLong()
     val step = input.readLong()
     val end = input.readLong()
     val spreadInt = input.readInt()
     val spread = if (spreadInt == -1) None else Some(spreadInt)
+    val queryPathString = input.readString()
+    val queryPath = if (queryPathString == "-1") None else Some(queryPathString)
     val procFailure = input.readBoolean()
-    PromQlQueryParams(config, promQl, start, step, end, spread, procFailure)
+    val procMultiPartition = input.readBoolean()
+    val verbose = input.readBoolean()
+    PromQlQueryParams(promQl, start, step, end, spread, queryPath, procFailure, procMultiPartition, verbose)
```

Comment:
We had config inside PromQlQueryParams before which was throwing NullPointerException when serializing. Since now PromQlQueryParams does not have config we should be able to remove it.